content/integrate/redis-data-integration/data-pipelines/data-denormalization.md

```diff
@@ -52,6 +75,7 @@ output:
           table: Invoice
         nesting_key: InvoiceLineId # cannot be composite
```

Comment:
nesting key is not explained well - it's the unique key of each entry in the json map that will be created under the `path` 
also parent_key and child_key are the columns used for the join